prototypes/styletransfer/tests/test_model.py

```diff
@@ -12,10 +12,22 @@ def _sample_img(size):
 def test_model():
     content_img = _sample_img(512)
     style_img = _sample_img(512)
-
+    init_img = _sample_img(512)
+   
     model = StyleTransfer()
 
-    content_rep, style_rep = model.feature_representations(content_img, style_img)
+    style_rep, content_rep = model.feature_representations(content_img, style_img)
```

Comment:
Oh so it works even when `content_img` and `style_img` are rank 3 tensors (instead of rank 4 where the batch size is the first dimension?)

Reply:
from `feature_representations` above it seems to return `content_rep, style_rep`